mod-circulation/src/main/resources/domain/mod-circulation/circulation-junit.feature

```diff
@@ -62,6 +62,7 @@ Feature: mod-circulation integration tests
       | 'owners.item.post'                                             |
       | 'users.item.post'                                              |
       | 'usergroups.item.post'                                         |
+      | 'inventory.items.item.mark-restricted.post'                    |
```

Comment:
Please sort a permissions based on alphabet

Reply:
Sure, it is done. Thanks